Create hamburger menu for Navbar

dfizzkruah93 opened the pull request:
- Created a Navbar with a hamburger menu

efoosag committed:
add background hovering effect to button

button turns green on hover

dfizzkruah93 committed:
Update README.md

dfizzkruah93 committed:
Add google analytics to index file and update CSS

dfizzkruah93 committed:
Merge branch 'main' of github.com:dfizzkruah93/Pofolio-Project into animation-transition

dfizzkruah93 committed:
Update README file and style sheet

dfizzkruah93 committed:
Update README file

dfizzkruah93 committed:
Update index and css file

dfizzkruah93 committed:
Merge branch 'animation-transition' of github.com:dfizzkruah93/Pofolio-Project into animation-transition

dfizzkruah93 committed:
Linters file configuration

tkz96 committed:
Fixed Linter Errors

tkz96 committed:
Create hamburger menu without functionality

tkz96 committed:
Add --fix in linters

tkz96 committed:
Remove --fix from Webhint

dfizzkruah93 committed:
Update Stylesheet for hamburger menu

Update index file

Add Javascript to enabled interactive and added image for hambuger menu

prolajumokeoni commented on index.html:
```diff
@@ -1,295 +1,394 @@
+        <div class="blur">
+          <div id="mobile-exit" class="mobile-menu-exit" onclick="mobileMenu()">
+            <img src="img/exit.svg" alt="Close" />
+          </div>
```
- [ ] Kindly align the logo  and hamburger menu icon on the mobile menu to look like the original design 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![fig](https://user-images.githubusercontent.com/69638013/157225656-2c80df5c-deb8-455a-9e7d-af7027a29e9a.PNG) | ![image](https://user-images.githubusercontent.com/69638013/157225770-64d52c0e-7187-4b3f-b471-796c82e0580f.png)

- [ ] When I click any of the  mobile links it reloads, please change this so when a user clicks on a link it takes them to the correct path on the page
- [ ]  There is currently a horizontal scroll on your website, a user has to scroll to the right on the page to look for the close button, please fix this as it makes your design and layout better
- [ ] It would be better to adjust the animations in the header so it does not disrupt your design
- Kindly see the video below ⬇️ 

https://user-images.githubusercontent.com/69638013/157224762-9b3d45d7-a1eb-429a-a72b-aa8f7be52d49.mp4

prolajumokeoni reviewed (changes requested):
Hi @dfizzkruah93 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow correctly used
✔️ Good design

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dfizzkruah93 committed:
Fix mobile link navigation

remove horizontal scroll

adjust annimation in header

align logo and hamburger menu

dfizzkruah93 committed:
Fix linters error

dfizzkruah93 committed:
Fix linters error

dfizzkruah93 commented:
@prolajumokeoni . Please help me review the changes that I have made as was indicated.

Kingstalux reviewed (approved):
Hi @team,
# Status: Approved ✔️ ✔️ ✔️ 
 
![128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c](https://user-images.githubusercontent.com/85113567/157292447-3a02fe7c-2ad6-473d-9bd4-cd4adb7aa65b.gif)


Great job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL) It would be great if you could add a section to your README file explaining how a user can set up and run your project locally on their machine. This will make your README file more professional 💯 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dfizzkruah93 commented:
@Kingstalux.  Thanks for the feedback and approval

dfizzkruah93 committed:
Edit README file
